sdk/clientcore/core/src/main/java/io/clientcore/core/implementation/http/rest/SwaggerMethodParser.java

```diff
@@ -148,6 +149,38 @@ public SwaggerMethodParser(Method swaggerMethod) {
             }
         }
 
+        final String[] requestQueryParams = httpRequestInformation.queryParams();
+
+        if (requestQueryParams != null) {
+            for (final String queryParam : requestQueryParams) {
+                final int equalsIndex = queryParam.indexOf("=");
+
+                if (equalsIndex >= 0) {
+                    final String paramName = queryParam.substring(0, equalsIndex).trim();
+
+                    if (!paramName.isEmpty()) {
+                        String paramValue = queryParam.substring(equalsIndex + 1).trim();
+
+                        if (!paramValue.isEmpty()) {
```

Comment:
Query parameter values should be allowed to be empty and having leading and trailing spaces.